src/components/IncorporationAgreement/AgreementType.vue

```diff
@@ -1,69 +1,66 @@
 <template>
-  <div>
-    <div id="agreement-summary" v-if="isSummary">
-      <v-card flat>
-        <!-- Summary Header -->
-        <div class="agreement-summary-header">
-          <v-icon color="dkBlue">mdi-handshake</v-icon>
-          <label class="agreement-summary-title">
-            <strong>Incorporation Agreement and {{getEntityDescription}} Articles</strong>
-          </label>
-        </div>
-        <!-- Summary Warning -->
-        <div
-          v-if="showErrorSummary"
-          class="agreement-invalid-message invalid-section"
-        >
-          <span>
-            <v-icon color="error">mdi-information-outline</v-icon>
-            &nbsp;
-            <span class="error-text">This step is unfinished.</span>
-            &nbsp;
-            <router-link
-              id="router-link"
-              :to="{ path: `/${RouteNames.INCORPORATION_AGREEMENT}` }"
-            >Return to this step to finish it</router-link>
-          </span>
-        </div>
-
-        <!-- Summary Content -->
-        <div v-else class="summary-desc">
-          <div><v-icon color="green darken-2" class="agreement-valid-icon">mdi-check</v-icon></div>
-            <div v-html="agreementTypeDescription"></div>
-        </div>
-      </v-card>
+  <v-card flat id="agreement-summary" class="rounded-0">
```

Comment:
I used Vuetify helper class "rounded-0" to remove the border radius, which made the red validation bar look weird. I think the v-cards look fine like this.